httpstan/services_stub.py

```diff
@@ -133,10 +135,16 @@ async def call(
                 # no messages right now and not done. Sleep briefly so other pending tasks get a chance to run.
                 await asyncio.sleep(0.001)
 
-    for fh in messages_files.values():
-        fh.flush()
-    httpstan.cache.dump_fit(b"".join(fh.getvalue() for fh in messages_files.values()), fit_name)
-    for fh in messages_files.values():
-        fh.close()
+    # WISHLIST: Here we compress messages after they all have arrived. Find a way to compress
+    # messages as they arrive.  Compressing messages as they arrive would use much less memory.
+    with lz4.frame.LZ4FrameCompressor() as compressor:
```

Comment:
The compress level by default is 0 (lowest compression).

Any idea how increasing compression changes compression time vs compression rate?

What is enough, like sending data over http vs compressing on cpu.

Reply:
I think our choice is between no compression and lowest/fastest compression. Compression slows things down. And speed is a bigger problem for us right now. We're still much slower than CmdStan.